### ISSUE TITLE
ImageNet 512x512 training config

### ISSUE THREAD

**Issue by KimSoybean** [sihyun-yu/REPA#35]
Hello, thanks for your great work.

Recent I have seen your update arxiv version with ImageNet 512x512 training.

But I have not found the detailed config in updated version, such as learning rate, noise scheduler, batchsize and other details.

Could you please provide those training details about 512x512 ?

**Comment by sihyun-yu** [comment]
Hi, it is currently included in our README file.

**Comment by KimSoybean** [comment]
Is that means that the bs=32*8?
